<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="2" failed="2" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-14T19:51:43 PST" name="Surefire suite" finished-at="2022-11-14T19:53:31 PST" duration-ms="108136">
    <groups>
      <group name="ErrorHandling">
        <method signature="ErrorValidationTest.submitWrongLogin()[pri:0, instance:com.ecommerce.testcases.ErrorValidationTest@51f116b8]" name="submitWrongLogin" class="com.ecommerce.testcases.ErrorValidationTest"/>
      </group> <!-- ErrorHandling -->
      <group name="Purchase">
        <method signature="SubmitOrderTest.submitOrder(java.util.HashMap)[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" name="submitOrder" class="com.ecommerce.testcases.SubmitOrderTest"/>
      </group> <!-- Purchase -->
    </groups>
    <test started-at="2022-11-14T19:51:43 PST" name="Surefire test" finished-at="2022-11-14T19:53:31 PST" duration-ms="108136">
      <class name="com.ecommerce.testcases.SubmitOrderTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" started-at="2022-11-14T19:52:34 PST" name="launchApplication" finished-at="2022-11-14T19:52:40 PST" duration-ms="5556" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" started-at="2022-11-14T19:52:40 PST" name="submitOrder" data-provider="getData" finished-at="2022-11-14T19:52:49 PST" duration-ms="9555" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Iamking@000, email=anshika@gmail.com, productName=ZARA COAT 3}]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="CloseApplication()[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" started-at="2022-11-14T19:52:49 PST" name="CloseApplication" finished-at="2022-11-14T19:53:05 PST" duration-ms="16068" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" started-at="2022-11-14T19:53:05 PST" name="launchApplication" finished-at="2022-11-14T19:53:11 PST" duration-ms="5552" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitOrder(java.util.HashMap)[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" started-at="2022-11-14T19:53:11 PST" name="submitOrder" data-provider="getData" finished-at="2022-11-14T19:53:15 PST" duration-ms="4139" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[{password=Iamking@000, email=anshika@gmail.com, productName=ADIDAS ORIGINAL}]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.ElementClickInterceptedException">
            <message>
              <![CDATA[element click intercepted: Element <button _ngcontent-ivl-c33="" routerlink="/dashboard/cart" class="btn btn-custom" style="margin-top: -10px;" tabindex="0">...</button> is not clickable at point (1668, 27). Other element would receive the click: <div _ngcontent-ivl-c31="" class="ngx-spinner-overlay ng-tns-c31-1 ng-trigger ng-trigger-fadeIn ng-star-inserted ng-animating" style="background-color: rgba(51, 51, 51, 0.8); z-index: 99999; position: fixed;">...</div>
  (Session info: chrome=107.0.5304.107)
Build info: version: '4.6.0', revision: '79f1c02ae20'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [0bc99458e2f4445a494b7f608c2734e0, clickElement {id=75fa548c-cb1e-49e0-93e3-14f203657e9f}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 107.0.5304.107, chrome: {chromedriverVersion: 107.0.5304.62 (1eec40d3a576..., userDataDir: C:\Users\swiss\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:59893}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:59893/devtoo..., se:cdpVersion: 107.0.5304.107, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (0bc99458e2f4445a494b7f608c2734e0)] -> css selector: [routerlink*='cart']]
Session ID: 0bc99458e2f4445a494b7f608c2734e0]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <button _ngcontent-ivl-c33="" routerlink="/dashboard/cart" class="btn btn-custom" style="margin-top: -10px;" tabindex="0">...</button> is not clickable at point (1668, 27). Other element would receive the click: <div _ngcontent-ivl-c31="" class="ngx-spinner-overlay ng-tns-c31-1 ng-trigger ng-trigger-fadeIn ng-star-inserted ng-animating" style="background-color: rgba(51, 51, 51, 0.8); z-index: 99999; position: fixed;">...</div>
  (Session info: chrome=107.0.5304.107)
Build info: version: '4.6.0', revision: '79f1c02ae20'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '11.0.7'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Command: [0bc99458e2f4445a494b7f608c2734e0, clickElement {id=75fa548c-cb1e-49e0-93e3-14f203657e9f}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 107.0.5304.107, chrome: {chromedriverVersion: 107.0.5304.62 (1eec40d3a576..., userDataDir: C:\Users\swiss\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:59893}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: WINDOWS, proxy: Proxy(), se:cdp: ws://localhost:59893/devtoo..., se:cdpVersion: 107.0.5304.107, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Element: [[ChromeDriver: chrome on WINDOWS (0bc99458e2f4445a494b7f608c2734e0)] -> css selector: [routerlink*='cart']]
Session ID: 0bc99458e2f4445a494b7f608c2734e0
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:200)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:133)
at org.openqa.selenium.remote.codec.w3c.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:53)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:184)
at org.openqa.selenium.remote.service.DriverCommandExecutor.invokeExecute(DriverCommandExecutor.java:167)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:142)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:541)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:257)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:78)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:52)
at com.sun.proxy.$Proxy14.click(Unknown Source)
at com.ecommerce.abstractcomponents.AbstractComponents.goToCartPage(AbstractComponents.java:55)
at com.ecommerce.testcases.SubmitOrderTest.submitOrder(SubmitOrderTest.java:27)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.ElementClickInterceptedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitOrder -->
        <test-method is-config="true" signature="CloseApplication()[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" started-at="2022-11-14T19:53:15 PST" name="CloseApplication" finished-at="2022-11-14T19:53:31 PST" duration-ms="16066" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
        <test-method signature="orderHistory()[pri:0, instance:com.ecommerce.testcases.SubmitOrderTest@625732]" started-at="2022-11-14T19:53:31 PST" name="orderHistory" finished-at="2022-11-14T19:53:31 PST" depends-on-methods="com.ecommerce.testcases.SubmitOrderTest.submitOrder" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.ecommerce.testcases.SubmitOrderTest.orderHistory() on instance com.ecommerce.testcases.SubmitOrderTest@625732 depends on not successfully finished methods 
[com.ecommerce.testcases.SubmitOrderTest.submitOrder() on instance com.ecommerce.testcases.SubmitOrderTest@625732]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.ecommerce.testcases.SubmitOrderTest.orderHistory() on instance com.ecommerce.testcases.SubmitOrderTest@625732 depends on not successfully finished methods 
[com.ecommerce.testcases.SubmitOrderTest.submitOrder() on instance com.ecommerce.testcases.SubmitOrderTest@625732]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- orderHistory -->
      </class> <!-- com.ecommerce.testcases.SubmitOrderTest -->
      <class name="com.ecommerce.testcases.ErrorValidationTest">
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.ecommerce.testcases.ErrorValidationTest@51f116b8]" started-at="2022-11-14T19:51:43 PST" name="launchApplication" finished-at="2022-11-14T19:51:50 PST" duration-ms="6994" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="errorProductValidation()[pri:0, instance:com.ecommerce.testcases.ErrorValidationTest@51f116b8]" started-at="2022-11-14T19:51:50 PST" name="errorProductValidation" finished-at="2022-11-14T19:51:54 PST" duration-ms="4192" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at com.ecommerce.testcases.ErrorValidationTest.errorProductValidation(ErrorValidationTest.java:33)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:135)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:193)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:94)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:146)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- errorProductValidation -->
        <test-method is-config="true" signature="CloseApplication()[pri:0, instance:com.ecommerce.testcases.ErrorValidationTest@51f116b8]" started-at="2022-11-14T19:51:54 PST" name="CloseApplication" finished-at="2022-11-14T19:52:10 PST" duration-ms="16070" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
        <test-method is-config="true" signature="launchApplication()[pri:0, instance:com.ecommerce.testcases.ErrorValidationTest@51f116b8]" started-at="2022-11-14T19:52:10 PST" name="launchApplication" finished-at="2022-11-14T19:52:17 PST" duration-ms="6496" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchApplication -->
        <test-method signature="submitWrongLogin()[pri:0, instance:com.ecommerce.testcases.ErrorValidationTest@51f116b8]" started-at="2022-11-14T19:52:17 PST" name="submitWrongLogin" finished-at="2022-11-14T19:52:18 PST" duration-ms="1093" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- submitWrongLogin -->
        <test-method is-config="true" signature="CloseApplication()[pri:0, instance:com.ecommerce.testcases.ErrorValidationTest@51f116b8]" started-at="2022-11-14T19:52:18 PST" name="CloseApplication" finished-at="2022-11-14T19:52:34 PST" duration-ms="16068" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CloseApplication -->
      </class> <!-- com.ecommerce.testcases.ErrorValidationTest -->
    </test> <!-- Surefire test -->
  </suite> <!-- Surefire suite -->
</testng-results>
